platforms/jvm/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -500,6 +514,7 @@ public void setDebug(boolean enabled) {
      * {@inheritDoc}
      */
     @Override
+    @ToBeKeptEagerProperty(because = "Read-only nested property")
```

Comment:
❗ Shouldn't need annotation as it's `@Nested`.

Reply:
Done, removed annotation `@ToBeKeptEagerProperty`. `@Nested` is inherited so this annotation is not needed anymore